src/scss/components/_select.scss

```diff
@@ -40,6 +59,18 @@
   &:focus { outline: none; }
 }
 
+.blip-select__arrow-down {
+  visibility: hidden;
+  width: 20px;
+  min-width: 20px;
```

Comment:
Is this necessary?

Reply:
Yes. If this is not defined, the arrow is not centralized in the input.